src/main/webapp/welcome.js

```diff
@@ -125,11 +125,40 @@ function setWelcomeMessage_(userName) {
 //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 // CREATE MAP OR USER
 
+/**
+ * Hides the parent node the given element is in
+ * @param {Element} nodeChild the child node of the element to be hidden
+ */
+function hideNode(nodeChild) {
```

Comment:
A better name would be `hideParentNode` since you're not directly hiding the argument.

Reply:
Fixed!